### PR TITLE
Move from URL span handling to clickable span, addresses issue #1

### PR DESCRIPTION
**Pull request by madhu314**

### CODE REVIEW

File: better-link-movement-method/src/main/java/me/saket/bettermovementmethod/BetterLinkMovementMethod.java [hunk 1]
```diff
@@ -305,13 +306,46 @@ protected void removeUrlHighlightColor(TextView textView) {
         Selection.removeSelection(text);
     }
 
-    protected void dispatchUrlClick(TextView textView, URLSpan span) {
-        final String spanUrl = span.getURL();
+    protected void dispatchUrlClick(TextView textView, ClickableSpanWithText spanWithText) {
+        final String spanUrl = spanWithText.text();
```
Comment (Saketme) [inline better-link-movement-method/src/main/java/me/saket/bettermovementmethod/BetterLinkMovementMethod.java]: Can you please rename this to `spanURL`?

Comment (madhu314) [inline better-link-movement-method/src/main/java/me/saket/bettermovementmethod/BetterLinkMovementMethod.java]: Will do

File: better-link-movement-method/src/main/java/me/saket/bettermovementmethod/BetterLinkMovementMethod.java [hunk 2]
```diff
@@ -34,17 +35,17 @@
 
     private OnLinkClickListener onLinkClickListener;
     private final RectF touchedLineBounds = new RectF();
-    private boolean isUrlHighlighted;
+    private boolean isURLHighlighted;
```
Comment (JakeWharton) [inline better-link-movement-method/src/main/java/me/saket/bettermovementmethod/BetterLinkMovementMethod.java]: I would revert all these uppercasing changes. Acronyms should be cased like words. See https://google.github.io/styleguide/javaguide.html#s5.3-camel-case

Comment (Saketme) [inline better-link-movement-method/src/main/java/me/saket/bettermovementmethod/BetterLinkMovementMethod.java]: Fixed.

I was casing acronyms like `URL` in uppercase to keep my code consistent with `URLSpan`.

**Pull request merged**